src/scaffold/ScaffoldView.tsx

```diff
@@ -107,87 +107,101 @@ const FirstPage: React.FC<PageProps> = ({ onNext }) => {
         </div>
       </div>
       <div className="btnContainer">
-        <VSCodeButton
-          className="navigation-btn"
-          id="next"
-          direction="next"
-          href="#"
-          disabled={!canGoNext}
-          onClick={next}>
+        <VSCodeButton className="navigation-btn" href="#" disabled={!canGoNext} onClick={next}>
           Next
         </VSCodeButton>
       </div>
     </>
   );
 };
 
-const SecondPage: React.FC<PageProps> = ({ onPrevious }) => {
+const SecondPage: React.FC<PageProps> = ({ onPrevious, options, setOptions }) => {
   const { state, actions } = useStore();
-  const [selectedBundler, setSelectedBundler] = React.useState<string | undefined>(undefined);
+
+  const { repoType, template, name, bundler, targetFolder } = options;
+  const canScaffold = repoType !== '' && template !== '' && name !== '' && bundler !== '' && targetFolder !== '';
+
+  const openLocalPathModal = () => {
+    actions.selectLocalPath();
+  };
+
+  const createPiralPilet = () => {
+    actions.scaffold(options);
+  };
+
+  const setLocalPath = (value: string) => {
+    setOptions((options) => ({ ...options, targetFolder: value }));
+  };
+
+  React.useEffect(() => {
+    setOptions((options) => ({ ...options, targetFolder: state.localPath }));
+  }, [state.localPath]);
 
   return (
     <>
       <div className="container">
         <div className="second-container">
           <div className="containerInfos">
-            <div className="informations">
+            <div className="information">
               <p className="columnTitle">Provide Information</p>
               <div className="extraItem">
-                <p className="extraItemLabel">
-                  Name <span className="errorName errorMessage hide">[required]</span>
-                </p>
-                <VSCodeTextField className="extraItemInput" stateName="name" />
+                <p className="extraItemLabel">Name</p>
+                <VSCodeTextField
+                  className="extraItemInput"
+                  stateName="name"
+                  value={options.name}
+                  onKeyUp={(ev: any) => setOptions((options) => ({ ...options, name: ev.target.value }))}
+                />
               </div>
               <div className="extraItem">
-                <VSCodeTextField id="local-path-input" className="extraItemInput" stateName="targetFolder">
-                  <p className="extraItemLabel">
-                    Local Path <span className="errorLocalPath errorMessage hide">[required]</span>
-                  </p>
-                  <span slot="end" id="local-path">
+                <VSCodeTextField
+                  className="extraItemInput"
+                  value={options.targetFolder}
+                  onKeyUp={(ev: any) => setLocalPath(ev.target.value)}>
```

Comment:
Why not use `onChange` here? `onKeyUp` is not really good - especially since you can right-click and click on paste.